### PR TITLE
Icanhazlesstmpltsplz

### PR DESCRIPTION
**Pull request by heffergm**
- common template for services that can use it
- fix bogus attribute ([:valhalla][:config_dir] doesn't exist)

Limited testing! I've verified the services start, but I'm not up on the whole workflow now, so please double check things still function as expected.

### CODE REVIEW

File: templates/default/sv-proxyd-global-run.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-cd <%= node[:valhalla][:config_dir] %>
+cd <%= node[:valhalla][:conf_dir] %>
 PROXY_IN=$(jq -r ".loki.service.proxy" <%= node[:valhalla][:conf_dir] %>/<%= node[:valhalla][:config] %>)_in
```
Comment (kevinkreiser) [inline templates/default/sv-proxyd-global-run.erb]: this has to change based on the layer. notice where it says loki, it should say the layer. maybe this:

```
PROXY_IN=$(jq -r ".<%= @options[:layer] %>.service.proxy" <%= node[:valhalla][:conf_dir] %>/<%= node[:valhalla][:config] %>)_in
PROXY_OUT=$(jq -r ".<%= @options[:layer] %>.service.proxy" <%= node[:valhalla][:conf_dir] %>/<%= node[:valhalla][:config] %>)_out
```

**Comment by kevinkreiser** [comment]
if you make the change that i commented above it should work. thanks for this massive cleanup. i figured there was a better way but was on a self imposed deadline ;o)

**Comment by kevinkreiser** [comment]
so yeah after change, above ship

**Comment by heffergm** [comment]
:ship: :it:

**Pull request merged**